doc-content/kogito-docs/src/main/asciidoc/openshift/chap-kogito-deploying-on-openshift.adoc

```diff
@@ -1230,12 +1230,170 @@ On Spring Boot, `spring.data.mongodb.uri` does not work simultaneously with host
+
+.Procedure
+
+. Create a `<filename.yaml>` YAML file containing the following template:
++
+----
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  name: <broker-name>
+----
++
+The YAML file creates a broker with the desired name in the current namespace. For more information about configuring broker options using YAML, see https://knative.dev/docs/eventing/broker/example-mtbroker/[Broker configuration example - Knative].
+
+. To apply the YAML file, enter the following command:
++
+----
+kubectl apply -f <filename>.yaml>
+----
++
+Where `<filename>` is the name of the file you created in the previous step.
+
+. Optional: To verify that the broker is working correctly, enter the following command:
++
+----
+kubectl -n <namespace> get broker <broker-name>
+----
++
+The information about your broker is displayed:
++
+If the broker is working correctly, it shows a *READY* status of `True`, for example:
++
+[cols="15%,15%,15%,40%,15%", options="header"]
+|===
+|NAME
+|READY
+|REASON
+|URL
+|AGE
+
+|<broker-name>
+|True
+|
+|http://broker-ingress.knative-eventing.svc.cluster.local/event-example/<broker-name>
+|1m
+|===
++
+If the *READY* status is `False`, wait a few moments and then run the command again.
+
+==== Binding Knative Broker with {PRODUCT} operator
+
+. Create your {PRODUCT} project locally. For more information about Serverless Workflow, see https://docs.jboss.org/kogito/release/latest/html_single/#chap-kogito-orchestrating-serverless[Orchestrating microservices with Serverless Workflow in Kogito].
+. Build the image with your project. For an example project, see https://github.com/kiegroup/kogito-examples/tree/stable/kogito-quarkus-examples/process-knative-quickstart-quarkus[Process with Knative Eventing] on GitHub.
+. Deploy the project as either a Knative Service or a regular deployment.
+. Create the {PRODUCT} Source CR, for example:
++
+----
+apiVersion: kogito.knative.dev/v1alpha1
+kind: KogitoSource
+metadata:
+  name: my-kogito-source
+spec:
+  subject:
+    apiVesion: serving.knative.dev/v1
+    kind: Service
+    # Put the name of your custom Kogito Service [3]
+    name: my-kogito-service
+  sink:
+    ref:
+      apiVersion: eventing.knative.dev/v1
+      kind: Broker
+      name: <broker-name>
+----
+. To deploy your source, enter the following command:
++
+----
+kubectl apply -f <path to the source>.yaml
+----
+
+The custom Kogito Service is now configured to produce cloud events that will be collected by the broker.
+
+If you want the custom Kogito Service to consume Knative Events you can use Knative triggers. For more information, see https://docs.jboss.org/kogito/release/latest/html_single/#con-knative-eventing_kogito-developing-process-services[Knative Eventing in Kogito Services].
+
+==== Using the Kogito Knative Add-on
+
+You can use the {PRODUCT} Add-on instead of manually configuring definitions of the YAML file descriptors for your custom {PRODUCT} service.
```

Comment:
```suggestion
You can use the {PRODUCT} add-on instead of manually configuring definitions of the YAML file descriptors for your custom {PRODUCT} service.
```

Reply:
Done